lua/ii/txi.lua

```diff
@@ -13,11 +13,43 @@ do return
     }
   }
 , getters =
-  { { name = 'value'
-    , cmd  = 1
+  { { name = 'in'
+    , cmd  = 0x0
```

Comment:
Cool! Will check this. Can we give them better names anyway? Would it be better to call:
`ii.txi.get('knob')` and `ii.txi.get('cv')`?

Reply:
I believe when I looked into this I saw that the “param” and “in” were the terms used for those values in the teletype firmware. I thought it would generally be good to use those same names in order to stay in step with anyone who was already a txi owner and any documentation that might be out there alread.